setup.py

```diff
@@ -53,8 +51,9 @@
         install_requires=install_requirements,
         extras_require={
             "test": test_requirements,
-            "data": data_requires,
-            "dev": test_requirements + data_requires,
+            "data": data_requirements,
+            "cluster": cluster_requirements,
+            "dev": test_requirements + data_requirements + cluster_requirements,
```

Comment:
Can you look at what was done here (https://github.com/ihmeuw/vivarium_research_template/pull/60/files#diff-00deb1ddb83583a1419a7b9cb7f80f60b1bee81d53b70a313d4a5cc27a8200a7) and match this. 

As you have it written, `[dev]` will include `data`, which means we will not end up with an environment using pandas 2